src/components/MultiHopTrade/components/TradeConfirm/hooks/useTradeSteps.tsx

```diff
@@ -0,0 +1,98 @@
+import { useMemo } from 'react'
+import { selectIsActiveQuoteMultiHop } from 'state/slices/tradeInputSlice/selectors'
+import {
+  selectActiveQuote,
+  selectHopExecutionMetadata,
+} from 'state/slices/tradeQuoteSlice/selectors'
+import { useAppSelector } from 'state/store'
+
+import {
+  countTradeSteps,
+  getCurrentTradeStep,
+  getCurrentTradeStepIndex,
+  getTradeSteps,
+} from '../helpers'
+import { useCurrentHopIndex } from './useCurrentHopIndex'
+
+export const useTradeSteps = () => {
```

Comment:
See comment re: trade steps terminology above, though nothing that requires immediate change!